languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt

```diff
@@ -35799,4 +35799,80 @@ linguicide;linguicide;N m s
 linguicides;linguicide;N m p
 locker;locker;N m s
 lockers;locker;N m p
-Dnipro;Dnipro;Z e sp
\ No newline at end of file
+Dnipro;Dnipro;Z e sp
```

Comment:
Yes, I didn't think about N as a possibility, but it makes sense :)

@jaumeortola do you think it would be better to add them to multi-token including the determiners? The original list I have already includes them, and we do have some entries in the multi-token file that include determiners

Reply:
Also, what are the real impact of the Z versus N tags: is it only a distinction used, in the end, in XML rules?

Reply:
I am not sure. Maybe add them to both places: added.txt and multiwords.txt with the determiner. 
If the tags are correct, it is not necessary to add them to multiwords.txt. (?)
If the determiner is mandatory, you will need a rule.

Reply:
Thank you, alright then, if we all agree, I'll proceed as follows:
- Change from Z to N
- I'm not adding them to multiwords.txt since the tags are correct
- Determiners are mandatory here (almost 100% sure), we can consider having a rule (but the most important is to stop flagging them as spelling errors, so it would be low prio)